### PR TITLE
Support for Sass and LESS with CSS Modules

### PR DESCRIPTION
**Pull request by jestho**
Hey Satyajit!

Here's my PR for Sass and LESS w/ CSS Modules. I skipped Stylus as I'm not very familiar with it.
Works with hot reloading, and seems to be bundling just fine.

I took the liberty to modify the template a bit - hope that's ok.

### CODE REVIEW

File: src/webpack-config.js [hunk 1]
```diff
@@ -35,6 +35,14 @@ module.exports = {
             {
                 test: /\.json$/,
                 loader: 'json-loader'
+            },
+            {
+                test: /\.less$/,
+                loader: 'style!css?modules&importLoaders=2&sourceMap&localIdentName=[local]___[hash:base64:5]!autoprefixer?browsers=last 2 version!less?outputStyle=expanded&sourceMap'
```
Comment (satya164) [inline src/webpack-config.js]: Can you use an array here? I think it's more readable.

```js
loaders: [
    'style-loader',
    'css-loader?modules&importLoaders=2&sourceMap&localIdentName=[local]___[hash:base64:5]',
    'autoprefixer-loader?browsers=last 2 version',
    'less-loader?outputStyle=expanded&sourceMap'
]
```

**Comment by satya164** [comment]
Thanks a lot for this. Just one thing I mentioned and we'll merge :)

**Comment by jestho** [comment]
Don't know if you get notified by new commits (I'm pretty new to contributing on Github), but I've changed the loaders to use arrays now :smile:

**Comment by satya164** [comment]
@jestho Thanks. No GH doesn't notify for new commits :(

**Comment by jestho** [comment]
:tada:

**Pull request merged**